tools/storage-bench/put.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"crypto/rand"
+	"fmt"
+	mrand "math/rand"
+	"os"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/codegangsta/cli"
+	"github.com/coreos/etcd/storage"
+)
+
+func newPutWorkload() cli.Command {
+	return cli.Command{
+		Name:      "put",
+		Usage:     "putting bunch of keys",
+		ArgsUsage: " ",
+		Flags: []cli.Flag{
+			cli.IntFlag{Name: "nr-keys", Value: 1000, Usage: "a number of keys to put"},
+			cli.IntFlag{Name: "key-size", Value: 64, Usage: "a size of key (Byte)"},
+			cli.IntFlag{Name: "value-size", Value: 64, Usage: "a size of value (Byte)"},
+			cli.BoolFlag{Name: "txn", Usage: "put a key in transaction or not"},
+		},
+		Action: handlePutWorkload,
+	}
+}
+
+func createBytesSlice(bytesN, sliceN int) [][]byte {
+	rs := [][]byte{}
```

Comment:
`rs := make([][]byte, sliceN)
for i := range rs {
  rs[i] = make([]byte, bytesN)
  ...
}
`